pkg/controllers/node/suite_test.go

```diff
@@ -285,6 +286,27 @@ var _ = Describe("Controller", func() {
 			node = ExpectNodeExists(ctx, env.Client, node.Name)
 			Expect(node.DeletionTimestamp.IsZero()).To(BeFalse())
 		})
+		It("should requeue reconcile if node is empty, but not past emptiness TTL", func() {
+			provisioner.Spec.TTLSecondsAfterEmpty = ptr.Int64(30)
+			currentTime := time.Now()
+			injectabletime.Now = func() time.Time { return currentTime } // injectabletime.Now() is called multiple times in function being tested.
```

Comment:
Kinda related to the below comment - I needed to do this because I had to repeat the math in the test since we lose precision in the timestamp conversions.